File: src/types/BlogPost.ts
export type BlogPostDTO = {
  id: string;
  date: Date;
  blogText: string;
  imageAlt: string;
  keyChanges: string[];
  gameId: string;
};

export interface BlogEntryData {
  date: string | Date;
  blogText: string;
  image: string;
  imageAlt: string;
  keyChanges: string[];
}

File: Backend.Api/Models/BlogPost.cs
namespace Backend.Api.Models;

public class BlogPost
{
    public Guid Id { get; set; }
    public DateTime Date { get; set; } = DateTime.UtcNow;
    public string BlogText { get; set; } = string.Empty;
    public string ImageAlt { get; set; } = string.Empty;
    public string[] KeyChanges { get; set; } = Array.Empty<string>();
    public Guid GameId { get; set; }
    public string Image { get; set; } = string.Empty;
}